--- shared.gpr.in.old	2007-06-13 11:36:51.000000000 +0200
+++ shared.gpr.in	2007-06-07 21:53:17.578125000 +0200
@@ -7 +7 @@
-   Build : Build_Type := External ("Build", "Debug");
+   Build : Build_Type := External ("Build", "Production");
--- templates_parser/templates_parser.gpr	(revision 170830)
+++ templates_parser/templates_parser.gpr	(working copy)
@@ -38,7 +38,7 @@
    for Library_Kind use "static";
 
    type Build_Type is ("Debug", "Release");
-   Build : Build_Type := external ("PRJ_BUILD", "Debug");
+   Build : Build_Type := external ("PRJ_BUILD", "Release");
 
    Processors := External ("PROCESSORS", "2");
 
