--- shared.gpr.old	2007-06-13 11:36:51.000000000 +0200
+++ shared.gpr	2007-06-07 21:53:17.578125000 +0200
@@ -7 +7 @@
-   Build : Build_Type := External ("Build", "Debug");
+   Build : Build_Type := External ("Build", "Production");
--- gnatlib/gnatcoll_shared.gpr.in.old	2007-06-13 11:35:36.828125000 +0200
+++ gnatlib/gnatcoll_shared.gpr.in	2007-06-13 11:34:49.656250000 +0200
@@ -5 +5 @@
-   Build : Build_Type := External ("Build", "Debug");
+   Build : Build_Type := External ("Build", "Production");
--- templates_parser/tp_shared.gpr.old	2008-05-20 16:12:51.062500000 +0200
+++ templates_parser/tp_shared.gpr	2008-05-20 16:14:04.015625000 +0200
@@ -33,3 +33,3 @@
    type Build_Type is ("Debug", "Release");
-   Build : Build_Type := external ("PRJ_BUILD", "Debug");
+   Build : Build_Type := external ("PRJ_BUILD", "Release");
 
