*** shared.gpr.old	Tue Oct  1 18:21:51 2002
--- shared.gpr	Tue Oct  1 18:22:03 2002
***************
*** 4,10 ****
     OS : OS_Type := External ("OS", "unix");
  
     type Build_Type is ("Debug", "Production");
!    Build : Build_Type := External ("Build", "Debug");
  
     for Source_Dirs use ();
  
--- 4,10 ----
     OS : OS_Type := External ("OS", "unix");
  
     type Build_Type is ("Debug", "Production");
!    Build : Build_Type := External ("Build", "Production");
  
     for Source_Dirs use ();
  
--- gnat_src/ali.ads	5 May 2006 13:17:47 -0000	1.124
+++ gnat_src/ali.ads	17 May 2006 12:34:20 -0000
@@ -38,6 +38,9 @@
 
 package ALI is
 
+   No_Restrictions : Restrictions_Info;
+   --  Stub for GPS compilation: unused by GPS.
+
    --------------
    -- Id Types --
    --------------
--- gnat_src/targparm.ads	22 Mar 2006 11:15:19 -0000	1.58
+++ gnat_src/targparm.ads	17 May 2006 12:34:27 -0000
@@ -81,6 +81,9 @@
 
 package Targparm is
 
+   No_Restrictions : Restrictions_Info;
+   --  Stub for GPS compilation: unused by GPS.
+
    ---------------------------
    -- Configuration Pragmas --
    ---------------------------
--- gnat_src/osint.adb	4 May 2006 03:23:56 -0000	1.335
+++ gnat_src/osint.adb	17 May 2006 12:34:36 -0000
@@ -735,7 +735,7 @@
       end if;
 
       if Executable_Extension_On_Target = No_Name then
-         Exec_Suffix := Get_Target_Executable_Suffix;
+         Exec_Suffix := Get_Executable_Suffix;
       else
          Get_Name_String (Executable_Extension_On_Target);
          Exec_Suffix := new String'(Name_Buffer (1 .. Name_Len));
--- gnat_src/osint.ads	21 Mar 2006 02:10:35 -0000	1.143
+++ gnat_src/osint.ads	17 May 2006 12:34:39 -0000
@@ -561,7 +561,7 @@
    --  No_File, that indicates that the file whose name was returned by the
    --  last call to Next_Main_Source (and stored here) is to be read.
 
-   Target_Object_Suffix : constant String := Get_Target_Object_Suffix.all;
+   Target_Object_Suffix : constant String := Get_Object_Suffix.all;
    --  The suffix used for the target object files
 
    Output_FD : File_Descriptor;
--- gnat_src/prj-nmsc.adb	8 Apr 2006 02:10:41 -0000	1.174
+++ gnat_src/prj-nmsc.adb	17 May 2006 12:35:08 -0000
@@ -60,7 +60,7 @@
    ALI_Suffix   : constant String := ".ali";
    --  File suffix for ali files
 
-   Object_Suffix : constant String := Get_Target_Object_Suffix.all;
+   Object_Suffix : constant String := Get_Object_Suffix.all;
    --  File suffix for object files
 
    type Name_Location is record
