--- gdk/x11/gdkdisplay-x11.c.old	Tue Feb 25 19:06:50 2003
+++ gdk/x11/gdkdisplay-x11.c	Tue Feb 25 19:08:10 2003
@@ -129,9 +129,6 @@
   display_x11->use_xft = -1;
   display_x11->xdisplay = xdisplay;
 
-  /* Set up handlers for Xlib internal connections */
-  XAddConnectionWatch (xdisplay, gdk_internal_connection_watch, NULL);
-  
   /* initialize the display's screens */ 
   display_x11->screens = g_new (GdkScreen *, ScreenCount (display_x11->xdisplay));
   for (i = 0; i < ScreenCount (display_x11->xdisplay); i++)
@@ -243,14 +240,6 @@
 			     GIOCondition cond,
 			     gpointer     data)
 {
-  GdkInternalConnection *connection = (GdkInternalConnection *)data;
-
-  GDK_THREADS_ENTER ();
-
-  XProcessInternalConnection ((Display*)connection->display, connection->fd);
-
-  GDK_THREADS_LEAVE ();
-
   return TRUE;
 }
 
--- modules/input/gtkimcontextxim.c.orig	Thu Jan 30 22:18:12 2003
+++ modules/input/gtkimcontextxim.c	Tue Feb 25 19:10:23 2003
@@ -247,6 +247,7 @@
 static void
 setup_im (GtkXIMInfo *info)
 {
+#if 0
   XIMValuesList *ic_values = NULL;
 
   if (info->im == NULL)
@@ -301,6 +302,7 @@
 
   if (ic_values)
     XFree (ic_values);
+#endif
 }
 
 static void
@@ -682,6 +684,7 @@
 static void
 gtk_im_context_xim_reset (GtkIMContext *context)
 {
+#if 0
   GtkIMContextXIM *context_xim = GTK_IM_CONTEXT_XIM (context);
   XIC ic = gtk_im_context_xim_get_ic (context_xim);
   gchar *result;
@@ -737,6 +740,7 @@
     }
 
   XFree (result);
+#endif
 }
 
 /* Mask of feedback bits that we render
